superdesk/media/renditions.py

```diff
@@ -403,7 +406,10 @@ def transfer_renditions(renditions, request_kwargs=None):
                 rend["href"] = app.media.url_for_media(rend["media"], local.content_type)
                 continue
 
-        content, filename, content_type = download_file_from_url(rend.get("href"), request_kwargs)
+        if feeding_service is not None and getattr(feeding_service, "download_file"):
+            content, filename, content_type = feeding_service.download_file(rend.get("href"), **request_kwargs or {})
+        else:
+            content, filename, content_type = download_file_from_url(rend.get("href"), request_kwargs)
```

Comment:
those 2 sections with `if feeding service ...` seems like duplicates, could be extracted to another method